src/extractor.ts

```diff
@@ -39,51 +42,65 @@ import { createInternalLogger } from './internal-loger'
 import groupBy from 'lodash.groupby'
 import camelcase from 'lodash.camelcase'
 
+interface ServeFontStubResponse {
+  extension: Format
+  content: Buffer
+  id: string
+}
 export default function FontExtractor (pluginOption: PluginOption = { type: 'auto' }): Plugin {
   const mode: PluginOption['type'] = pluginOption.type ?? 'manual'
   let cache: Cache | null
   let importResolvers: ImportResolvers
   let logger: InternalLogger
 
   let isServe: boolean = false
-  const fontServeProxy = new Map<string, {
-    extension: Format
-    content: Buffer
-  }>()
-
-  const autoTarget: Required<Target> = {
-    fontName: '[auto detected font name]',
-    raws: [],
-    withWhitespace: true,
-    ligatures: [],
-  }
+  const fontServeProxy = new Map<string, () => Promise<ServeFontStubResponse | null>>()
+
+  const glyphsFindMap = new Map<string, string[]>()
+
+  const autoTarget = new Proxy<Required<Target>>(
+    {
+      fontName: '[auto font target have not current font name]',
```

Comment:
maybe you need more informative message?

Reply:
Changed